docs/messaging/nats-component.md

```diff
@@ -54,6 +52,42 @@ public class ExampleService(INatsConnection client)
 }
 ```
 
+## App host usage
+
+To model the Nats resource in the app host, install the [Aspire.Hosting.Nats](https://www.nuget.org/packages/Aspire.Hosting.Nats) NuGet package.
+
+### [.NET CLI](#tab/dotnet-cli)
+
+```dotnetcli
+dotnet add package Aspire.Hosting.Nats
+```
+
+### [PackageReference](#tab/package-reference)
+
+```xml
+<PackageReference Include="Aspire.Hosting.Nats"
+                  Version="[SelectVersion]" />
+```
+
+---
+
+Then, in the _:::no-loc text="Program.cs":::_ file of `AppHost`, register a NATS server and consume the connection using the following methods:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var nats = builder.AddNats("nats");
+
+var myService = builder.AddProject<Projects.MyService>()
+                       .WithReference(nats);
+```
+
+The `WithReference` method configures a connection in the `MyService` project named `nats`. In the _:::no-loc text="Program.cs":::_ file of `MyService`, the NATS connection can be consumed using:
```

Comment:
You could set Program.cs as file-level, folder-level, or global no-loc metadata.

Reply:
Yes, that's a good idea. Thank you!